app/src/main/java/org/schabi/newpipe/util/DrawableResolver.kt

```diff
@@ -0,0 +1,26 @@
+package org.schabi.newpipe.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import android.util.TypedValue
+import androidx.annotation.AttrRes
+import androidx.core.content.ContextCompat
+
+/**
+ * Utility class for resolving [Drawables](Drawable)
+ */
+object DrawableResolver {
```

Comment:
Why is this here and not in `ThemeHelper`? I'd prefer not to create yet another class in the `util` package when there is already one that contains similar functions

Reply:
``ThemeHelper`` is used for doing theme-related stuff (e.g. ``getThemeForService``). ``DrawableResolver`` is used for resolving Drawables.

Not sure how these classes should be related...

Reply:
No, `ThemeHelper` contains things that have to do with theme and Context resources. Those two things are tightly coupled since the resources are determined by the theme. Hence why, for example, `resolveColorFromAttr` is in that class, and `resolveDrawable` should go there too. Before #5927 `resolveResourceIdFromAttr` existed in that class, too.